federation/pkg/kubefed/init/init_test.go

```diff
@@ -1418,7 +1418,7 @@ func tlsHandshake(t *testing.T, sCfg, cCfg *tls.Config) error {
 	// clients hang. Since TCP is non-blocking we use that as transport
 	// instead. One could have as well used a Unix Domain Socket, but TCP is
 	// more portable.
-	s, err := tls.Listen("tcp", "", sCfg)
+	s, err := tls.Listen("tcp4", "", sCfg) // specify tcp4 for the guys who disable ipv6
```

Comment:
`who have enabled IPv6` you mean in your comment?

Also, can you put that comment above this line?